subgraph/src/FundingRoundFactoryMapping.ts

```diff
@@ -13,11 +13,13 @@ import {
 import { MACIFactory as MACIFactoryContract } from '../generated/FundingRoundFactory/MACIFactory'
 import { FundingRound as FundingRoundContract } from '../generated/FundingRoundFactory/FundingRound'
 
-import { OptimisticRecipientRegistry as RecipientRegistryContract } from '../generated/FundingRoundFactory/OptimisticRecipientRegistry'
+import { OptimisticRecipientRegistry } from '../generated/FundingRoundFactory/OptimisticRecipientRegistry'
+import { PessimisticRecipientRegistry } from '../generated/FundingRoundFactory/PessimisticRecipientRegistry'
 
 import {
   FundingRound as FundingRoundTemplate,
-  OptimisticRecipientRegistry as recipientRegistryTemplate,
+  OptimisticRecipientRegistry as optimisticRecipientRegistryTemplate,
+  PessimisticRecipientRegistry as pessimisticRecipientRegistryTemplate,
```

Comment:
@daodesigner wondering if you know how we could do this.

Basically, how we could create the correct registry template: check this line https://github.com/ethereum/clrfund/blob/develop/subgraph/src/FundingRoundFactoryMapping.ts#L121

Is there a way we can detect the registry type before the creation of the template?
```
# something like this
if recipientRegistry === 'optimistic'
	create optimistic template
if recipientRegistry === 'pessimistic'
	create pessimistic template
```

but I don't know if that is possible. Our only data at this point is the `recipientRegistryAddress`.